patches/server/0035-Disable-thunder-and-add-chance-config.patch

```diff
@@ -1,33 +1,40 @@
 From 0000000000000000000000000000000000000000 Mon Sep 17 00:00:00 2001
 From: Sudzzy <originmc@outlook.com>
 Date: Wed, 2 Mar 2016 14:52:43 -0600
-Subject: [PATCH] Disable thunder
+Subject: [PATCH] Disable thunder and add chance config
 
 
 diff --git a/src/main/java/com/destroystokyo/paper/PaperWorldConfig.java b/src/main/java/com/destroystokyo/paper/PaperWorldConfig.java
-index cac8606f65e587d34e2ae12f5962f236c316751e..b18622c199f57e0d7c201582fe83bb75dca143f9 100644
+index cac8606f65e587d34e2ae12f5962f236c316751e..a039979d44a78cf86c99f2d8ac448887ff1cd328 100644
 --- a/src/main/java/com/destroystokyo/paper/PaperWorldConfig.java
 +++ b/src/main/java/com/destroystokyo/paper/PaperWorldConfig.java
-@@ -171,4 +171,9 @@ public class PaperWorldConfig {
+@@ -171,4 +171,16 @@ public class PaperWorldConfig {
      private void disableExplosionKnockback(){
          disableExplosionKnockback = getBoolean("disable-explosion-knockback", false);
      }
 +
 +    public boolean disableThunder;
 +    private void disableThunder() {
 +        disableThunder = getBoolean("disable-thunder", false);
++    }
++
++    public int thunderChance;
++    private void thunderChance() {
++        thunderChance = getInt("thunder-chance", 100000);
++
++        log("Thunder has a 1/" + thunderChance + " chance of striking each chunk.");
```

Comment:
Move this into the disableThunder method, maybe rename that. Math.max the value so it can't be negative or 0, and no log message is needed here